commands/test.go

```diff
@@ -277,7 +277,7 @@ func (t *testcmd) runremotetest(ff *common.FuncFile, in *common.InputMap, expect
 	}
 	var stdout bytes.Buffer
 
-	if err := client.CallFN(t.provider, t.remote, ff.Path, stdin, &stdout, "", envVars, "application/json", false); err != nil {
```

Comment:
We should probably add a test for invoking functions here.

Reply:
tested